src/penn_chime/models.py

```diff
@@ -64,21 +64,40 @@ def __init__(self, p: Parameters):
             logger.info('Using doubling_time: %s', p.doubling_time)
 
             intrinsic_growth_rate = get_growth_rate(p.doubling_time)
-
             self.beta = get_beta(intrinsic_growth_rate,  gamma, self.susceptible, 0.0)
             self.beta_t = get_beta(intrinsic_growth_rate, self.gamma, self.susceptible, p.relative_contact_rate)
 
-            self.i_day = 0 # seed to the full length
-            raw = self.run_projection(p, [(self.beta, p.n_days)])
-            self.i_day = i_day = int(get_argmin_ds(raw["census_hospitalized"], p.current_hospitalized))
+            if p.mitigation_date is None:
+                self.i_day = 0 # seed to the full length
+                raw = self.run_projection(p, [(self.beta, p.n_days)])
+                self.i_day = i_day = int(get_argmin_ds(raw["census_hospitalized"], p.current_hospitalized))
 
-            self.raw = self.run_projection(p, self.gen_policy(p))
+                self.raw = self.run_projection(p, self.gen_policy(p))
+
+                logger.info('Set i_day = %s', i_day)
+            else:
+                projections = {}
+                best_i_day = -1
+                best_i_day_loss = float('inf')
+                for i_day in range(p.n_days):
+                    self.i_day = i_day
+                    raw = self.run_projection(p, self.gen_policy(p))
+
+                    # Don't fit against results that put the peak before the present day
```

Comment:
This seems to be a no-op? Can we get rid of it?

Reply:
Play with the parameters more - those cases do come up

Reply:
Sorry totally misread this. LGTM